konfera/views.py

```diff
@@ -13,3 +14,20 @@ def sponsor_list_view(request, event_slug):
     return render(request=request,
                   template_name='konfera/event_sponsors.html',
                   context=context, )
+
+
+def event_list(request):
+    events = Event.objects.all().order_by('date_from')
+    paginator = Paginator(events, 2)
```

Comment:
2 is good for testing but I think we should show more of them on a single page (of course it depends on the design which we don't have for now)